src/backend/distributed/citus--7.5-3--7.5-4.sql

```diff
@@ -0,0 +1,47 @@
+/* citus--7.5-3--7.5-4 */
+
+CREATE FUNCTION pg_catalog.citus_query_stats(OUT queryid bigint,
+											 OUT userid oid,
+											 OUT dbid oid,
+											 OUT executor bigint,
+											 OUT partition_key text,
+											 OUT calls bigint)
+RETURNS SETOF record
+LANGUAGE C STRICT
+AS 'MODULE_PATHNAME', $$citus_query_stats$$;
+
+CREATE FUNCTION pg_catalog.citus_stat_statements_reset()
+RETURNS VOID
+LANGUAGE C STRICT
+AS 'MODULE_PATHNAME', $$citus_stat_statements_reset$$;
+
+CREATE FUNCTION pg_catalog.citus_stat_statements(OUT queryid bigint,
+												 OUT userid oid,
+												 OUT dbid oid,
+												 OUT query text,
+												 OUT executor bigint,
+												 OUT partition_key text,
+												 OUT calls bigint)
+RETURNS SETOF record
+LANGUAGE plpgsql
+AS $citus_stat_statements$
+BEGIN
+ IF EXISTS (
+ 	SELECT name FROM pg_extension
```

Comment:
this gives me an error:
```
ERROR:  column "name" does not exist
LINE 2:   SELECT name FROM pg_extension
```

Should be `SELECT extname`, though since we don't care about the value, event `SELECT 1` is sufficient.